src/AirshipBindings.iOS/ApiDefinitions.cs

```diff
@@ -2436,10 +2506,12 @@ interface UARegistrationDelegate
 
         // - (void)registrationSucceededForChannelID:(nonnull NSString *)channelID deviceToken:(nonnull NSString *)deviceToken;
         [Export("registrationSucceededForChannelID:deviceToken:")]
+        [Obsolete("Deprecated ? to be removed in SDK version 13.0. Please use the UAChannelUpdatedEvent NSNotification.")]
```

Comment:
these question marks seem problematic. I'm guessing this is the unicode thing you were talking about?

Reply:
Yup. Double dash gets converted into question mark by the C# utility. I'll manually fix them up.